tests/image-stream/utils.spec.js

```diff
@@ -25,29 +25,44 @@ describe('ImageStream: Utils', function() {
 
   describe('.getArchiveMimeType()', function() {
 
-    it('should return application/x-bzip2 for a bz2 archive', function() {
+    it('should resolve application/x-bzip2 for a bz2 archive', function(done) {
       const file = path.join(DATA_PATH, 'bz2', 'raspberrypi.img.bz2');
-      m.chai.expect(utils.getArchiveMimeType(file)).to.equal('application/x-bzip2');
+      utils.getArchiveMimeType(file).then((type) => {
```

Comment:
I took the chance to make this function async. I'll update the commit.

> Does the getArchiveMimeType docstring need updating too?

Oops, good catch :)